aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -141,81 +142,20 @@ the edge of the screen.
   <PageContainerFull />
 </Example>
 
-Use cases include:
+**Use cases may include:**
```

Comment:
The use of "may" here is a subtle difference between the others that are more definitive. I am worried people will either miss the difference all together and assume that a table should use the "full" page or will wonder "When is it okay to use a table as full width"? 

We can enhance this in future, but I think we should either give some elaboration here for each bullet point such as "complex tables that would otherwise require lots of horizontal scrolling" or link to a section with the appropriate guidance.

Reply:
Adding this comment ☝️  to the Figma file so it doesn't get lost when this PR is merged.